config/locales/en/loan_questions.en.yml

```diff
@@ -29,6 +29,23 @@ en:
     range_of_cells: Optional cells for Google spreadsheet
     url: URL
 
+  loan_report:
```

Comment:
What is a "Loan Report"? Isn't it just the loan memo? If so, perhaps the below strings should go under an existing key?

Also I don't think loan_questions is the right file for this. I think loan_questions is the area where you define the questions, not where you answer them or view the answers.

I think the appropriate place would be loans.en.yml. There is some existing questionnaire stuff in there.